.github/workflows/android-unit-tests.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         working-directory: apps/expo-go/android
         run: ./gradlew spotlessCheck || { echo '::error Spotless lint failed. Run `./gradlew spotlessApply` from `apps/expo-go/android` to automatically fix formatting.' && exit 1; }
       - name: 🎸 Run native Android unit tests
+        if: always()
```

Comment:
If the spotless check fails, but native unit tests pass, will the job finish as failed or succeeded?

Reply:
failed

Reply:
So I'm ok with this specific change 👍

Reply:
also fine for this change given ubuntu runners have more availability.